x/liquidstakeibc/client/tx.go

```diff
@@ -157,6 +158,36 @@ func NewLiquidStakeCmd() *cobra.Command {
 	return cmd
 }
 
+func NewLiquidStakeCmdLSM() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "liquid-stake-lsm [delegations]",
+		Short: `Liquid Stake an existing delegation from a registered host chain into stk tokens`,
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			clientctx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			var delegations []*sdk.Coin
```

Comment:
why json, it's just cli, use strings here like msgSendCoin, 100uxprt,100uatom,...

Reply:
its being used on the other commands too